pyproject.toml

```diff
@@ -1,3 +1,9 @@
+[tool.codespell]
+skip = ".git,*.pdf,*.svg"
+# the way to make case sensitive skips of words etc
+ignore-regex = "\bNd\b"
+ignore-words-list = "te,crate"
```

Comment:
These should be replaced with inline comments once https://github.com/codespell-project/codespell/issues/1212 is fixed, but it looks like a global ignore is the best option at the moment.